lib/esm/generated/telemetry.d.ts

```diff
@@ -381,6 +381,10 @@ export declare type TelemetryConfigurationEvent = CommonTelemetryProperties & {
              * The list of events that include feature flags collection
              */
             collect_feature_flags_on?: ('view' | 'error' | 'vital')[];
+            /**
+             * Whether the anonymous users are tracked by extending cookies. Default to true
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
             * Whether the anonymous users are tracked by extending cookies.
```